data/changelog.txt

```diff
@@ -1,5 +1,22 @@
 # 0.F (2021-06-19)
 
+## Highlights
+Nested Containers rationalize inventory management and enable dropping and retrieving go-bags during fights.
+Achievements track your deeds and misdeeds across games.
+Proficiencies better represent deeper knowledge required for various endeavors, mostly crafting.
+Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment system.
+Weariness tracking added to represent longer term physical exhaustion.
+Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.
+Large scale audit of weapon and armor values for better representativeness and consistency.
+Improved armor handling by seperating ballistic damage into it's own damage type.
+Pervasive performance enhancements throughout the game.
+Tileset vehicle support for more cohesive vehicle rendering.
+Aftershock changing direction to a total conversion with a new far sci-fi setting set on a frozen world.
+Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, dino-related NPCs with missions and dino locations.
+Added many dino features, including zombie, fungal, evolved, bionic, baby, and mutant dino variants.
+Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
```

Comment:
```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update with many new traits called Attunements that switch up gameplay at the endgame.
```

Reply:
"Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

Reply:
"Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

Reply:
```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```

Reply:
Shouldn't each bullet that applies to a specific mod mention the mod? In the list at the top of this section Dinomod is mentioned for the first point, but I'd expect the two following ones to apply to the same mod only.

Reply:
```suggestion
Dinomod added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```